src/model/constraints/behavior_constraints.py

```diff
@@ -1,11 +1,5 @@
-from abc import abstractmethod
-from src.model.constraints.constraint import Constraint
+from src.model.constraints.constraint import Behavior_Con
 
-class Behavior_Con(Constraint):
-    @abstractmethod
-    def get_possible_paths(self): # TODO: Confirm naming and implement
-        return []
-    
 class Danger_Con(Behavior_Con):
     pass
```

Comment:
@ArenBabikian would you say `Danger_Con` should be an abstract class as well?

Reply:
@zhekai-jiang yes! this will make many things simpler from the perspective of the `complete` mode as well

Reply:
So I'll make it abstract and move it to constraint.py as well